doc/source/whatsnew/v2.2.0.rst

```diff
@@ -272,6 +272,25 @@ For example:
 
     pd.date_range('2020-01-01', periods=3, freq='QE-NOV')
 
+The alias ``Y`` is deprecated in favour of ``YE`` for offsets, please use ``YE`` for "year end" instead of ``Y`` (:issue:`54275`)
```

Comment:
Can we consolidate all these into a single message?

Something like:
```
Deprecated the following frequency aliases:
- `M` (month end) has been renamed `ME`
- ...
- `Y` (year end) has been renamed `YE`
```

[single example]

Reply:
thank you for the comment. I combined three notes into one and left one example for "quarter end".